xarray/tests/__init__.py

```diff
@@ -25,111 +40,32 @@
 except ImportError:
     import mock
 
-try:
-    import scipy
-    has_scipy = True
-except ImportError:
-    has_scipy = False
-
-try:
-    import pydap.client
-    has_pydap = True
-except ImportError:
-    has_pydap = False
-
-try:
-    import netCDF4
-    has_netCDF4 = True
-except ImportError:
-    has_netCDF4 = False
-
-
-try:
-    import h5netcdf
-    has_h5netcdf = True
-except ImportError:
-    has_h5netcdf = False
-
-
-try:
-    import Nio
-    has_pynio = True
-except ImportError:
-    has_pynio = False
-
+has_matplotlib, requires_matplotlib = _importorskip('matplotlib')
+has_scipy, requires_scipy = _importorskip('scipy')
+has_pydap, requires_pydap = _importorskip('pydap.client')
+has_netCDF4, requires_netCDF4 = _importorskip('netCDF4')
+has_h5netcdf, requires_h5netcdf = _importorskip('h5netcdf')
+has_pynio, requires_pynio = _importorskip('pynio')
+has_dask, requires_dask = _importorskip('dask')
+has_bottleneck, requires_bottleneck = _importorskip('bottleneck', '1.0')
```

Comment:
bottleneck 1.0 came out in Feb 2015. I'm not sure it's worth having the version check here.